lib/concurrent.rb

```diff
@@ -11,6 +11,7 @@
 require 'concurrent/executors'
 require 'concurrent/utilities'
 
+require 'concurrent/atomic/atomic_markable_reference'
```

Comment:
Since this will be part of the edge gem we'll need to move this "require" to `lib/concurrent-edge.rb`